draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.scss

```diff
@@ -16,20 +16,68 @@
 
 .header {
   color: $kz-color-wisteria-800;
-  padding: $ca-grid * 1.5 $ca-grid * 1.5 0;
+  padding: $ca-grid * 0.5 $ca-grid * 1.5;
+  text-align: center;
 }
 
-.body {
-  padding: $ca-grid $ca-grid * 1.5;
+.footer {
+  white-space: nowrap;
+  padding: $ca-grid 0 0 0;
 }
 
-.bodyWithoutFooter {
-  composes: body;
-  padding-bottom: $ca-grid * 2.5;
+.footerWithSecondaryAction {
+  composes: footer;
+  margin-left: -0.5 * $ca-grid;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex1 {
```

Comment:
I could be wrong, but it appears you are not using this class anywhere?

Reply:
ah yep a few there, one sec